apps/docs/pages/docs/api-reference/components/puck.mdx

```diff
@@ -25,21 +25,22 @@ export function Editor() {
-| [`ui`](#ui)                     | `ui: {leftSideBarVisible: false}`      | [AppState.ui](/docs/api-reference/app-state#ui)    | -            |
-| [`viewports`](#viewports)       | `viewports: [{ width: 1440 }]`         | [Viewport\[\]](#viewport-params)                   | -            |
+| Param                                   | Example                                           | Type                                               | Status       |
+| --------------------------------------- | ------------------------------------------------- | -------------------------------------------------- | ------------ |
+| [`config`](#config)                     | `config: { components: {} }`                      | [Config](/docs/api-reference/configuration/config) | Required     |
+| [`data`](#data)                         | `data: {}`                                        | [Data](/docs/api-reference/data)                   | Required     |
+| [`dnd`](#dnd)                           | `dnd: {}`                                         | [DndConfig](#dnd-params)                           | -            |
+| [`children`](#children)                 | `children: <Puck.Preview />`                      | ReactNode                                          | -            |
+| [`headerPath`](#headerpath)             | `headerPath: "/my-page"`                          | String                                             | -            |
+| [`headerTitle`](#headertitle)           | `headerTitle: "My Page"`                          | String                                             | -            |
+| [`iframe`](#iframe)                     | `iframe: {}`                                      | [IframeConfig](#iframe-params)                     | -            |
+| [`onChange()`](#onchangedata)           | `onChange: (data) => {}`                          | Function                                           | -            |
+| [`onPublish()`](#onpublishdata)         | `onPublish: async (data) => {}`                   | Function                                           | -            |
+| [`overrides`](#overrides)               | `overrides: { header: () => <div /> }`            | [Overrides](/docs/api-reference/overrides)         | Experimental |
+| [`plugins`](#plugins)                   | `plugins: [myPlugin]`                             | [Plugin\[\]](/docs/api-reference/plugin)           | Experimental |
+| [`ui`](#ui)                             | `ui: {leftSideBarVisible: false}`                 | [AppState.ui](/docs/api-reference/app-state#ui)    | -            |
+| [`viewports`](#viewports)               | `viewports: [{ width: 1440 }]`                    | [Viewport\[\]](#viewport-params)                   | -            |
+| [`initialHistories`](#initialHistories) | `initialHistories: [{ histories: [], index: 0 }]` | [Viewport\[\]](#initialHistories-params)           | -            |
```

Comment:
Thank you for doing the docs!

This table is actually alphabetical. Would you mind moving this under `iframe`?